src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -654,7 +654,7 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
     // These executions have no corresponding recorded operation ID and are therefore not resumable.
     def collectNonResumableDatabaseKeys(executionsAndJobs: Seq[(Execution, JesJob)]): Seq[ExecutionDatabaseKey] = {
       executionsAndJobs collect {
-        case (execution, job) if execution.status.toExecutionStatus == ExecutionStatus.Running && job.jesJobId.isEmpty =>
+        case (execution, job) if execution.status.toExecutionStatus == ExecutionStatus.Running && job.jesId.isEmpty =>
```

Comment:
I certainly wouldn't object to renaming this here and in the database, something like jesRunId maybe?